imports/api/eligibleClients/methods.js

```diff
@@ -23,6 +23,21 @@ Meteor.methods({
     return hc.api('house-matching-v2').getEligibleClientsPage(pageNumber, pageSize, sort, order);
   },
 
+  updateBonusScore(clientId, bonusScore) {
```

Comment:
We mostly have string-based method names i.e. 'module.function', so yours should be 'eligibleClients.updateBonusScore'